testing/tests/ui/blocks_below_top_level.rs

```diff
@@ -0,0 +1,36 @@
+use rinja::Template;
+
+#[derive(Template)]
+#[template(source = r#"
+{% block bla %}
+{% extends "bla.txt" %}
+{% endblock %}
+"#, ext = "txt")]
+struct MyTemplate1;
+
+#[derive(Template)]
+#[template(source = r#"
+{% block bla %}
+{% macro bla() %}
+{% endmacro %}
+{% endblock %}
+"#, ext = "txt")]
+struct MyTemplate2;
+
+#[derive(Template)]
+#[template(source = r#"
+{% block bla %}
+{% import "bla.txt" as blue %}
+{% endblock %}
+"#, ext = "txt")]
+struct MyTemplate3;
+
+#[derive(Template)]
```

Comment:
This might not be the best location for this test as it checks multiple `extend`s rather than blocks not at the top level

Reply:
Fair enough, moving it to its own file.